### ISSUE TITLE
Override completion broken

### ISSUE THREAD

**Issue by shmellyorc** [dotnet/vscode-csharp#6011]
No errors in vscode output but autocomplete broken again.

C# exntetion v2.0.320 for vscode 1.80.2. Using dotnet 7.0.306.

Doesnt matter if you press Tab, Click it in the list, or press enter or space, notthing happens:
![Code_abCNiakOS2](https://github.com/dotnet/vscode-csharp/assets/5901214/6f987265-d287-41ad-9e98-c3abe56f4de7)

**Comment by vladkor-alt** [comment]
понабирали ?%?(% по объявлению дебилов в команду разработки... походу нужно какой-то другой IDE искать...

**Comment by mkorsukov** [comment]
> понабирали ?%?(% по объявлению дебилов в команду разработки... походу нужно какой-то другой IDE искать...

Try this https://github.com/dotnet/vscode-csharp/issues/6016#issuecomment-1664305893

**Comment by dibarbet** [comment]
@genlu appears to be an override completion

**Comment by genlu** [comment]
I believe clicking on the override-completion item is a known issue with VSCode, though I couldn't find the bug you filed before @dibarbet 

But tab/enter should work. I tried this locally with VSCode 1.81 and C# extension 2.0.320 (w/o devkit) and it works as expected. Could you try repro this issue and share the related log with me? i.e. search for "completionItem/resolve" in the "C# LSP Trace Logs" tab of the output window. Something like 

```
[Trace - 2:09:54 PM] Received notification 'window/logMessage'.
Params: {
    "type": 4,
    "message": "[LanguageServerHost][09:09:54.747][Start]completionItem/resolve"
}


[Trace - 2:09:54 PM] Received response 'completionItem/resolve - (305)' in 9ms.
Result: {
    "label": "GetHashCode()",
    "kind": 2,
........
```

**Comment by rpahut** [comment]
Not sure who the request was directed at but in case it helps:
[5-C# LSP Trace Logs.portion.redacted.log](https://github.com/dotnet/vscode-csharp/files/12259562/5-C.LSP.Trace.Logs.portion.redacted.log)
The autocompletion here was supposed to insert _"void Initialize()\r\n    {\r\n        base.Initialize();\r\n    }"_ , have no idea what the other "bool Equals" or "int GetHashCode" blocks are.

**Comment by Fairpl3x** [comment]
got the same issue 
![258303944-a06247e2-0397-4226-b110-4c1db098bcd2](https://github.com/dotnet/vscode-csharp/assets/31825109/9907cc8b-9b65-4eb0-be9f-851302ee7ad2)

**Comment by luthfiampas** [comment]
Facing the same issue yesterday, it happens with XML comments suggestions as well. I've just updated to 1.81.1 and it works fine now. Don't know why.

**Comment by genlu** [comment]
There was a change in VSCode May release (I think 1.79?) that caused the problem with override and XML comment completion (see [this ](https://github.com/microsoft/vscode/issues/184924) issue for more details.) That change was reverted in June so the latest VSCode should work as expected. 

Could anyone who still experiences this issue try upgrade VSCode and see if it persist? Thanks!

**Comment by KCFindstr** [comment]
I'm on latest vscode (1.81) and still experiencing the override auto completion issue. I believe it's different from the XML one because I only had this problem after switching to the new c# extension (2.0+). The old extension with Roslyn analyzer was able to auto complete overrides as expected.

**Comment by dibarbet** [comment]
If anyone is repro'ing this on a project they can share publicly, please attach to this issue and also attach the entire contents of the folder specified by this command:
<img width="496" alt="image" src="https://github.com/dotnet/vscode-csharp/assets/5749229/f0639883-8042-46a7-abc3-dff04cafc64a">

**Comment by Fairpl3x** [comment]
> If anyone is repro'ing this on a project they can share publicly, please attach to this issue and also attach the entire contents of the folder specified by this command: <img alt="image" width="496" src="https://user-images.githubusercontent.com/5749229/260859602-f0639883-8042-46a7-abc3-dff04cafc64a.png">

I might be wrong but I think this ticked (closed by you btw) provides a way to reproduce the issue.

**Comment by JasonAtGitHub** [comment]
I'm having the same issue too.  I tried to disable **C# Dev Kit** and set the option `dotnet.server.useOmnisharp` to true, It worked as expected.  So I think the problem may lies in **C# Dev Kit**.

**Comment by dibarbet** [comment]
Note - the fix has been merged, but we have not been able to release a version with the fix.  We're hitting some other bugs that are delaying the release.  The current plan is to release next week.

**Comment by dibarbet** [comment]
We just released `2.3.27` of the C# extension ([changelog](https://github.com/dotnet/vscode-csharp/releases/tag/v2.3.27)) which has fixes for multiple bugs in override completion.

If you're still facing issues after upgrading, please open a *new* issue.  Thanks!
